packages/transport/src/fallback.ts

```diff
@@ -1,34 +1,26 @@
-// @ts-nocheck
-
-import type {
-    Transport,
-    AcquireInput,
-    TrezorDeviceInfoWithSession,
-    MessageFromTrezor,
-} from './types';
+import type { Transport, AcquireInput, TrezorDeviceInfoWithSession } from './types';
 
 export default class FallbackTransport {
-    name = `FallbackTransport`;
-    activeName = ``;
-
-    _availableTransports: Array<Transport>;
-    transports: Array<Transport>;
-    configured: boolean;
-    version: string;
-    debug = false;
-
-    // note: activeTransport is actually "?Transport", but
-    // everywhere I am using it is in `async`, so error gets returned as Promise.reject
+    _availableTransports: Array<Transport> = [];
+    activeName = '';
+    // @ts-ignore
```

Comment:
why is this needed?

Reply:
this variable can't be assigned in constructor and is accessed multiple times in this file. If I wanted to define type here it would need to be `Transport | undefined` and in that case I would have to do some runtime check to every place where this is used or do something like `this.activeTransport!.call()...`
I dont know which one is better.